core/src/main/java/org/apache/iceberg/FastAppend.java

```diff
@@ -143,12 +143,12 @@ private ManifestFile copyManifest(ManifestFile manifest) {
 
   @Override
   public List<ManifestFile> apply(TableMetadata base, Snapshot snapshot) {
-    List<ManifestFile> newManifests = Lists.newArrayList();
```

Comment:
What about renaming `writeManifests` to `writeNewManifests` to match `newManifests` variable and calling this variable simply `manifests` like we do in `MergingSnapshotProducer`?

Reply:
Updated.